extensions/amp-story/1.0/amp-story.js

```diff
@@ -1298,9 +1298,7 @@ export class AmpStory extends AMP.BaseElement {
     const uiState = this.getUIType_();
     this.storeService_.dispatch(Action.TOGGLE_UI, uiState);
 
-    if (uiState !== UIType.MOBILE ||
-        this.getSupportedOrientations_()
-            .includes(ScreenOrientations.LANDSCAPE)) {
+    if (uiState !== UIType.MOBILE || this.isLandscapeSupported_()) {
       this.storeService_.dispatch(Action.TOGGLE_LANDSCAPE, false);
```

Comment:
Can we add a comment / todo here please? It's still a bit confusing for me to read this.

Reply:
Done